# Request
int16 redu_ratio
---
# Response
bool redu_ratiograb
bool success
